command/untaint_test.go

```diff
@@ -0,0 +1,478 @@
+package command
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/mitchellh/cli"
+)
+
+func TestUntaint(t *testing.T) {
```

Comment:
Just a nitpick, but as this is a test testing noop case, can we also name it that way, so it's obvious?